engine/src/test/java/org/camunda/bpm/engine/test/api/task/TaskQueryTest.java

```diff
@@ -4812,6 +4813,35 @@ public void testQueryByNameNotLike() {
     assertEquals(11, query.count());
   }
 
+  @Deployment(resources = "org/camunda/bpm/engine/test/api/task/TaskQueryTest.testProcessDefinition.bpmn20.xml")
+  public void testQueryByProcessInstanceIdIn() {
```

Comment:
I think that we need a second test like this one, but with a third (existing) Process Instance. The purpose would be to assert that the new query parameter successfully filters Process Instances by their IDs.

The current tests show that a non-existent ID will not provide any additional Process Instances, however, there aren't any additional Process Instances in the database, to detect if the filter is failing.

Reply:
Please check https://github.com/camunda/camunda-bpm-platform/pull/618/files#diff-c41cc0cd1332900de7fca008a11cc53cR4821